topicgenerator/docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+#this file sets up elasticsearch, kibana, and postgres containers for running dockstore locally
+#dockstore can run  without elasticsearch but the search page requires elasticsearch to be running
+#kibana can be used alongside elasticsearch as a visual interface for the indices and send queries directly
+#this only brings up the postgres container, not the actual database
+version: '2.2'
+services:
+  es01:
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.10.1
+    container_name: es01
+    environment:
+      - node.name=es01
+      - cluster.name=es-docker-cluster
+      - discovery.type=single-node
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      # The max_content_length value on AWS OpenSearch is dependent on instance type
+      # https://docs.aws.amazon.com/opensearch-service/latest/developerguide/limits.html#network-limits
+      - http.max_content_length=10mb
+      - xpack.security.enabled=false
+      - xpack.monitoring.enabled=false
+      - xpack.graph.enabled=false
+      - xpack.watcher.enabled=false
+      - xpack.ml.enabled=false
+      - script.allowed_types= none
+      - script.allowed_contexts= none
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    volumes:
+      - data01:/usr/share/elasticsearch/data
+    ports:
+      - 9200:9200
+    networks:
+      - elastic
+      # uncomment to run a kibana instance
+      # kib01:
+      # image: docker.elastic.co/kibana/kibana:6.8.3
+      # container_name: kib01
+      # ports:
+      # - 5601:5601
+      # environment:
+      # ELASTICSEARCH_URL: http://es01:9200
+      # ELASTICSEARCH_HOSTS: http://es01:9200
+      # networks:
+      # - elastic
+  postgres_db:
+    image: postgres:16.1
```

Comment:
We're going to have a bunch of these scattered around, maybe a follow-up ticket to create a shared resource in https://github.com/dockstore/.github

This one in particular looks a lot like https://github.com/dockstore/dockstore-cli/blob/develop/docker-compose.yml with the right postgres version

Reply:
One idea, could download during build via https://github.com/maven-download-plugin/maven-download-plugin

Reply:
Created https://ucsc-cgl.atlassian.net/browse/SEAB-6352